.github/workflows/test.yml

```diff
@@ -214,7 +214,7 @@ jobs:
           # CPython
           - target: aarch64-unknown-linux-gnu
             arch: aarch64
-            abi: cp36-cp36m
+            abi: cp37-cp37m
```

Comment:
This is unfortunate. I don't think maturin is ready to drop cp36 abi3 support just yet since users can depend on old pyo3 versions.

@davidhewitt what do you think?

Reply:
Yeah I agree but was just trying to get it to pass the tests here for now. The pyo3-build-config dep could be optional but at that point I'm not sure it has the same benefit. Can the pyo3-build-config version be different from the pyo3 version used for the bindings?

Reply:
I'd make pyo3-build-config only does cross compiling detection for maturin and leave other stuffs to user's pyo3 dependency to decide.

Reply:
I couldn't find a link to the failed workflow run for cp36, what error does it gives when cross compiling for python3.6?

Reply:
It got stuck on this assert but I didn't chase down where/why that was being called.

https://github.com/aganders3/pyo3/blob/edb59c3be2793efc01aade80d1772bfe1217272b/pyo3-build-config/src/impl_.rs#L133

Reply:
I think it https://github.com/PyO3/pyo3/blob/5e8d43c8bd93ab6d0fe599307cc7508a368d7777/build.rs#L139

`build.rs` shouldn't run for maturin, I think we need a way to make it a no-op.

Reply:
Maybe it could toggle based on `CARGO_PRIMARY_PACKAGE`?

Reply:
Didn't know about `CARGO_PRIMARY_PACKAGE`, TIL! I'm not sure if it works, we can give it a try.

Reply:
> For example add a on-by-default feature to pyo3-build-config and disable it in maturin.

Actually I think this already exists with the `resolve-config` feature so I will try to disable it.

Reply:
Okay I pushed a change that uses an empty feature to disable this. I also found this in the docs on [Overriding Build Scripts](https://doc.rust-lang.org/cargo/reference/build-scripts.html#overriding-build-scripts) which seems like another (possibly more correct?) way to do this that would not require changes to pyo3 or changing the project Cargo.toml.

Reply:
Ah they way I did it also seems to break the non-cross-compiled case. I'll keep trying!